test/augmentation/test_augmentation.py

```diff
@@ -25,6 +25,334 @@
 )
 
 
+from kornia.testing import BaseTester
+from kornia.augmentation.base import AugmentationBase2D
+
+# TOTEST
+# _validate_input_dtype
+# _transform_input
+# identity_matrix
+
+# Common
+# Given arbitrary input shapes the output dims are consistent
+# shape changing not shape changing
+#  return_transform: bool = False, same_on_batch: bool = False, p: float = 0.5,
+#  p_batch: float = 1.
+# Smoke tests, lehet példányosítani, meg lehet hívni az abstract függvényeit stb
+# generate_parameters annyi paramot ad vissza amennyi a batch size
+# compute_transformation implementálva van és batchsiye stimmel
+# apply_transform implementálva van és batchsiye stimmel
+
+
+class CommonTests(BaseTester):
+
+    ############################################################################################################
+    # Attributes variables to set
+    ############################################################################################################
+    _augmentation_cls = None
+    _default_param_set = None
```

Comment:
I think this part is not that intuitive given ```param_set```. I think here, we may define a dummy example like 
```_augmentation = None```.  If need to initialize, you may do ```type(_augmentation)(**params)```.

Reply:
The concept behind, is that we want to create the augmentation objects within the test implementations to enforce certain behaviours by default (like p=0.0). _default_param_set is for having a default not trivial param set, that can be used for this.
As for param_set the name is a bit misleading, it's meant for the smoke tests to check if they can be run with every theoretically allowed arguments, I guess if we use them for every test that might be an overkill.

Reply:
I think it is better to avoid having two suites of params. I think the ```param_set``` shall be taken as including all the edge cases?

\cc @edgarriba